design/001-RPC-Interface-Specification.md

```diff
@@ -79,13 +77,17 @@ The following 3 data formats will be used in the APIs.
     * `Currency`: [String] Currency used for specifying the amount.
     * `Balance`: [Map of `Alias` to `Balance`]
       * `Alias`: [String] `Self` is a special alias for the user of the node.
-      * `Balances`: [String] Amount held by the `Peer` corresponding to the Alias.
+      * `Balances`: [String] Balance of the `Peer` corresponding to the Alias.
```

Comment:
This is unclear to me. Is the map's name supposed to be `Balances` and should the string representing a `Peer`'s balance be called `Balance` rather than the other way around?

Reply:
You are correct, it is more clear if map's name is `Balances` and the string field name is `Balance`. 
Updated.

Reply:
I have updated the format of payment channel info to use lists instead of maps. This has two advantages:
1. We require one term less. Instead of using `Balances` for the map and `Balance` for the field in previous version, now there is just one term `Balances` for the list.
2. When we add support for multiple assets (like other ERC20 tokens) to be held in the same channel, this structure can be scaled without repeating the list of aliases for each asset.

Reply:
I'd propose to use the singular 'Balance' for the list of amounts held by the channel participants, as the whole list is the channel's balance (invariant: sum of amounts).
Additionally, the constraints regarding the Alias and Balance lists should be documented here (lists have same length, values in Alias are unique, Alias is a known peer, and amount held by a particular peer whose Alias is Alias[idx] is balance[idx], amounts in balance are non-negative, ...)
Please add the rationale why you don't choose a map data structure mapping Alias to Amount but two lists.

Reply:
Okay, I have updated the spec to use the term `Balance` and have also documented the constraints and rationale for using two lists instead of a map.